SerialController/Commands/PythonCommands/SV_LocalRaid_0_9_1.py

```diff
@@ -279,6 +311,39 @@ def judge_raidPokemon_type(self):
         # 合致するものがいなかった場合水として返す(デフォルトはハラバリーを使用する)
         print("合致するタイプがないので水タイプとして処理します。")
         return Type.WATER
+    
+    def choose_raidPokemon_type(self, type: Type):
+        progress_raid_type = [
+                        Type.FIRE,
+                        Type.WATER,
+                        Type.ELECTRIC,
+                        Type.ICE, 
+                        Type.GROUND,
+                        Type.FLYING,
+                        Type.PSYCHIC,
+                        Type.BUG,
+                        Type.ROCK,
+                        Type.GHOST,
+                        Type.DRAGON,
+                        Type.STEEL
+                        ]
+        not_progress_raid_type = [
```

Comment:
```suggestion
        take_raid_type = [
                        Type.FIRE,
                        Type.WATER,
                        Type.ELECTRIC,
                        Type.ICE, 
                        Type.GROUND,
                        Type.FLYING,
                        Type.PSYCHIC,
                        Type.BUG,
                        Type.ROCK,
                        Type.GHOST,
                        Type.DRAGON,
                        Type.STEEL
                        ]
        skip_progress_raid_type = [
```
```suggestion
        take_raid_type = [
```

progress - not progressより、take - skipあたりの方が自然な気がする（takeよりもっと適切な語はありそうだけど、思いつかない。process？）

Reply:
なんか変になってたから修正しました。ごめんね

Reply:
proceed to raidが良さげらしい

Reply:
単純にexecute_raid_typeもいいかもしれない